ds_judgements_public_ui/sass/includes/_govuk_overrides.scss

```diff
@@ -186,3 +186,90 @@
 .govuk-form-group {
   margin-bottom: 0;
 }
+
+.govuk-header {
+  border-bottom: none;
+}
+
+.govuk-header__link--homepage:hover,
+.govuk-header__link--homepage:active {
+  border-bottom: $space-1 solid $color-black;
+}
+
+.govuk-header__navigation-item--active a,
+.govuk-header__navigation-item--active a:hover,
+.govuk-header__navigation-item--active a:visited {
+  color: $color-yellow;
+  font-weight: bold !important;
+
+  &:focus {
+    background-color: transparent;
+    box-shadow: none;
+  }
+}
+
+.govuk-header__navigation-item--active .govuk-header__link,
+.govuk-header__navigation-item .govuk-header__link:hover {
+  color: $color-yellow;
+  font-weight: bold;
+  border-bottom: $space-1 solid $color-yellow;
+  text-decoration: none;
+}
+
+.govuk-header__navigation-item .govuk-header__link {
+  font-size: 0.875rem;
+  line-height: $typography-md-line-height;
+  font-weight: normal;
+  white-space: nowrap;
+  text-decoration: none;
+  display: inline-block;
+
+  &:after {
+    display: block;
+    content: attr(title);
+    font-weight: bold;
+    height: 0;
+    overflow: hidden;
+    visibility: hidden;
+  }
+}
+
+a.govuk-header__link {
+  border-bottom: 1px solid $color-white;
+  @media only screen and (max-width: $grid-breakpoint-medium) {
+    border-bottom: 1px solid $color-black;
+  }
+}
+
+.govuk-header__menu-button:focus {
+  outline: $space-1 solid transparent;
+  color: #ffffff;
+  background-color: transparent;
+  text-decoration: none;
+  -webkit-box-decoration-break: clone;
+  box-decoration-break: clone;
+}
+
+.govuk-frontend-supported .govuk-header__menu-button {
+  right: $space-16;
```

Comment:
I've noticed when trying this out and clicking around on mobile it's very easy to click the logo instead of the menu as the clickable area for the logo is so big:

<img width="590" alt="image" src="https://github.com/nationalarchives/ds-caselaw-public-ui/assets/173052089/65001b96-4b32-407a-8796-bc51af62d27c">

(The blue shows the clickable area)

We should look at reducing that size on mobile.

Reply:
@jlhdxw thanks updated with an override to gov logo link sizing